EosPayload/drivers/ADC_driver.py

```diff
@@ -0,0 +1,93 @@
+import logging
+import time
+import traceback
+
+import Adafruit_BBIO.ADC as ADC
+
+from EosLib.packet.definitions import Device
+from EosPayload.lib.driver_base import DriverBase
+
+
+class ADCDriver(DriverBase):
+    ADC.setup()
```

Comment:
do not do this here!  override the setup function and do this in there -- see driver base for details on the EosPayload setup function